lib_xud/doc/rst/resource_usage_summary.rst

```diff
@@ -0,0 +1,28 @@
+Typical Resource Usage
```

Comment:
Why isn't resource_usage_summary.rst included anywhere?

Reply:
It was used only in the top-level README. 
In the lib_xud main documentation a more comprehensive file is used:
https://github.com/xmos/lib_xud/blob/develop/lib_xud/doc/rst/resource_usage.rst

As the top-level README is checked by the infr_apps scripts, we decided to move the details of "Typical Resource Usage" outside that file and to store them in a separate file inside the lib, in case someone wants to use it in the future.

Reply:
Understood; thank you for the explanation.